### PR TITLE
Pjg 70 regcontent template

### PR DESCRIPTION
**Pull request by gleckler1**
@durack1  pls review updates to issue template to include all registered content

### CODE REVIEW

**Comment by durack1** [comment]
@gleckler1 I have amended your PR to align with the new entries in #65 which is awaiting `CMOR 3.2.6` to be released.. If you wanted to convert the example to `RSS` solely (replace the PCMDI entry) be my guest - once you've checked this please merge the PR

**Comment by gleckler1** [comment]
@durack1  I don't understand.   I would like to use the ARC example, not PCMDI or RSS

**Comment by durack1** [comment]
Well the [ARC example is not currently registered](https://github.com/PCMDI/obs4MIPs-cmor-tables/blob/master/obs4MIPs_source_id.json), so would be a good idea to use a registered example..

**Comment by gleckler1** [comment]
more clear now

**Comment by gleckler1** [comment]
@durack1  I'd like to stick with the ARC example (its used in ODS), so I'm prepared to register it.   But we don't have the updated issue template ready (catch 22!).  Should I just make the mods to the writejson.py and submit them as a PR?

**Comment by durack1** [comment]
@gleckler1 yep, `writeJson.py` should work on crunch/ocean for you

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,22 +1,31 @@
 <Please fill out the requested information and delete irrelevant information from the template below before submitting your issue.>  
 
-To register (or edit) some or all of the obs4MIPs "registered content", please title this issue as follows:  
+To register (or edit) some or all of the obs4MIPs "registered content" (RC), please title this github issue as follows:
```
Comment (durack1) [inline .github/ISSUE_TEMPLATE.md]: (RC) is never used again, so no need to define it

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,22 +1,31 @@
 <Please fill out the requested information and delete irrelevant information from the template below before submitting your issue.>  
 
-To register (or edit) some or all of the obs4MIPs "registered content", please title this issue as follows:  
+To register (or edit) some or all of the obs4MIPs "registered content" (RC), please title this github issue as follows:  
 
-"registration information for [acronym describing your dataset - see source_id below]"
+"registration information for [acronym describing your dataset - see source_name below]"
```
Comment (durack1) [inline .github/ISSUE_TEMPLATE.md]: This says `source_name` whereas below it says `source_id`

**Pull request merged**